packages/flutter_tools/test/general.shard/ios/xcresult_test.dart

```diff
@@ -10,6 +10,7 @@ import 'package:flutter_tools/src/macos/xcode.dart';
 
 import '../../src/common.dart';
 import '../../src/fake_process_manager.dart';
+import 'xcresult_test_data.dart';
```

Comment:
Moving the data into a separate file to prepare for https://github.com/flutter/flutter/pull/94747.